ivy_tests/test_ivy/test_frontends/test_numpy/test_random/test_functions.py

```diff
@@ -203,3 +203,31 @@ def test_numpy_geometric(
         p=p,
         size=size,
     )
+
+
+# permutation
+@handle_frontend_test(
+    fn_tree="numpy.random.permutation",
+    dtype_and_x=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("numeric"),
+    ),
+)
+def test_numpy_permutation(
+    dtype_and_x,
+    size,
+    frontend,
+    test_flags,
+    fn_tree,
+    on_device,
+):
+    input_dtype, x = dtype_and_x
+    helpers.test_frontend_function(
+        input_dtypes=input_dtype,
+        frontend=frontend,
+        test_flags=test_flags,
+        fn_tree=fn_tree,
+        on_device=on_device,
+        x=x[0],
+        test_values=False,
+        size=size,
```

Comment:
indeed, as the tested function doesn't take any `size` argument, you should remove it from here and also from above :slightly_smiling_face: also probably good to move the `x=x[0]` to be the last argument (this won't make any difference but just general styling) :grin: